torch/onnx/_internal/exporter/_decomp.py

```diff
@@ -83,18 +81,14 @@ def create_onnx_friendly_decomposition_table(
         Dict[torch._ops.OperatorBase, Callable]: A dictionary that maps op overloads to their corresponding
         decomposition functions.
     """
-    decomposition_table: dict[torch._ops.OperatorBase, Callable] = {}
-
-    # NOTE: If we import torch._decomp, we will get RuntimeError: Only a single
-    # TORCH_LIBRARY can be used to register the namespace nvprims; please put all of your
-    # definitions in a single TORCH_LIBRARY block.
-    for op_overload, decomp_fn in torch._decomp.decomposition_table.items():  # type: ignore[attr-defined]
-        # Skip decomposition for op_overload as long as that op_overload has a corresponding ONNX
-        # symbolic function.
-        # NOTE: Do not skip torch._refs decomps. They are fine because otherwise the model is
-        # not exportable anyways.
-        if op_overload in onnx_registered_ops:
-            continue
-        decomposition_table[op_overload] = decomp_fn
+    decomposition_table: dict[torch._ops.OperatorBase, Callable] = (
+        torch._decomp.core_aten_decompositions()
```

Comment:
Is this list more complete than what is in `torch._decomp.decomposition_table`? I remember `torch._decomp.decomposition_table` had more decomps than `core_aten_decompositions`

Reply:
Technically decomposition_table has more decomps but in export context, all non-functional op decomps don't matter because we always functioanlize before decomp.

Reply:
I see. Just to confirm - I have seen prim ops in the previous table (so not core aten). Are they included now, or not needed?

Reply:
Ah good catch sorry. This logic seems bit wrong.

Reply:
core_aten don't include prim ops.